reference/employer/openapi.yaml

```diff
@@ -1036,6 +1036,63 @@ paths:
                     manual:
                       type: boolean
```

Comment:
i think we should be able to deprecate `manual` and `mfa_required` as well. now that we expose auth methods individually, manual = provider has the `assisted` auth method and `mfa_required` should only apply to a `credential` type
i had an example PR of that here: https://github.com/Finch-API/finch-docs/pull/187

Reply:
(i also think we need to deprecate `products` but that's probably a separate ticket)

Reply:
Sure I can take those out! Do you think we should version this change or it's not necessary?

Reply:
oh i just meant we can mark them as deprecated but not remove them just yet. we'd need to version to remove them
to mark `mfa_required` as deprecated we'd need to move that into the `authentication_methods` objects to denote whether its required for each individual auth method. not sure if that's a quick change or too much work and we can handle it separately
either way we can probably mark `manual` as deprecated right now, with similar text to what's in the PR i linked

Reply:
Sure thing. I'll just mark it as deprecated in the open API spec then!